docs/contributing/technical/setup-repo-with-git.md

```diff
@@ -1,44 +1,44 @@
 ---
 id: setup-repo-with-git
-title: Setting up a repository with Git and GitHub
-sidebar_label: Setting up a repository with Git and GitHub
+title: 'Setting Up a Repository with Git and GitHub'
+sidebar_label: Setting Up a Repository with Git and GitHub
 keywords:
-  - Setting up a repository with Git and GitHub
+  - setting up a repository with Git and GitHub
 ---
 
 ## Using the GitHub CLI
 
-### How to install the GitHub CLI
+### How to Install the GitHub CLI
 
 The [GitHub CLI](https://cli.github.com/) allows you to fork repositories, create issues, pull requests, and more from the command line.
 
-The GitHub CLI can be installed on Mac, Windows or Linux. You can find the complete installation instructions [here](https://github.com/cli/cli#installation).
+Follow these [instructions to install GitHub CLI](https://github.com/cli/cli#installation) on Mac, Windows, or Linux.
 
-### How to authenticate with the GitHub CLI
+### How to Authenticate with the GitHub CLI
 
 From the terminal, you will need to authenticate with the GitHub CLI. You can do this by running the following command:
 
 ```shell
 gh auth login
 ```
 
-Follow the on screen prompts to authenticate with the GitHub CLI.
+Then, follow the prompts to authenticate with the GitHub CLI.
 
-### How to fork and clone a repository with the GitHub CLI
+### How to Fork and Clone a Repository with the GitHub CLI
 
-A fork is a copy of a repository and it allows you to freely experiment with changes without affecting the original project.
+A fork is a remote copy of a repository, allowing you to experiment freely with changes without affecting the original project.
 
-A clone is a local copy of a repository that includes all the files, branches, and commits.
+A clone is a local copy of a repository that includes all the files, branches and commits.
 
 To fork and clone a repository with the GitHub CLI, run the following command:
 
 ```shell
-gh repo fork open-sauced/REPO_NAME
+gh repo fork open-sauced/<REPO_NAME>
 ```
 
 The GitHub CLI will fork the project in your GitHub account and will ask you if you want to clone the repository on your local machine.
```

Comment:
I'm not familiar with GitHub CLI and need confirmation on this. Most of resources I got is cloning a repo with [`gh repo clone`](https://cli.github.com/manual/gh_repo_clone) command.

Does this line mean that after forking a repo with CLI, it will give a prompt that asking if we want to clone it as well? 

```suggestion
The GitHub CLI will fork the project in your GitHub account and will give you a prompt asking if you want to clone the repository on your local machine.
```